src/views/PurchaseOrderDetail.vue

```diff
@@ -0,0 +1,172 @@
+<template>
+  <ion-page>
+    <ion-header :translucent="true">
+      <ion-toolbar>
+        <ion-back-button default-href="/" slot="start" />
+        <ion-title>{{$t("Purchase Order Details")}}</ion-title>
+        <ion-buttons slot="end">
+          <ion-button @click="receivingHistory">
+            <ion-icon :icon="timeOutline"/>
+          </ion-button>
+          <ion-button>
+            <ion-icon :icon="add"/>
+          </ion-button>
+        </ion-buttons>
+      </ion-toolbar>
+    </ion-header>
+
+    <ion-content :fullscreen="true">
+      <div>
+        <ion-item lines="none">
+          <h1>{{$t("Purchase Order")}}: PO10291</h1>
+        </ion-item>
+        <ion-item>
+          <ion-label>{{$t("Scan Items")}}</ion-label>
+          <ion-label>
+            <ion-input :placeholder="$t('Scan barcodes to receive')" />
+          </ion-label>
+        </ion-item>
+        <ion-card>
+          <div class="product-info">
+            <ion-item lines="none">
+              <ion-thumbnail slot="start">
+                <Image src="https://cdn.shopify.com/s/files/1/0069/7384/9727/products/test-track.jpg?v=1626255137" />
+              </ion-thumbnail>
+              <ion-label>
+                <ion-label>Chaz Kangeroo Hoodie-XS-Green</ion-label>
```

Comment:
Ion label cant be used inside ion-label. You have to write text without ion-label here.

Reply:
Written without ion-label